provision/kubernetes/variables.tf

```diff
@@ -51,3 +51,12 @@ variable "flavor" {
   default = "@FLAVOR"
 }
 
+variable "contrail_type" {
+  default = "vnc_api"
+}
+
+variable "patchsetRef" {
+  default = ""
```

Comment:
Maybe default patchset should be `master` for clarity?

Reply:
Done